javascript/big-practice/src/scripts/views/listNoteView.js

```diff
@@ -0,0 +1,82 @@
+import selectDOMClass from '../utils/selectDOMByClass';
+
+/**
+ * @class listNoteView
+ * @description manage view of listNote
+ */
+export default class ListNoteView {
+  constructor(noteView) {
+    this.noteView = noteView;
+    this.listNoteElement = selectDOMClass('.list-notes');
+
+    // input form
+    this.formElement = selectDOMClass('.form-add-note');
+    this.formTitleElement = selectDOMClass('.form-title');
+    this.formUtilitiesElement = selectDOMClass('.form-utilities');
+    this.closeButtonElement = selectDOMClass('.form-add-note .btn-close');
+    this.inputAddElement = selectDOMClass('.form-add-note .form-group-input .input-note');
+  }
+
+  /**
+   * @description function render all notes
+   *
+   * @param {Array} listNote is a list of notes from data
+   */
+  renderListNotes = (listNotes) => {
+    listNotes.forEach((note) => {
+      const noteItem = {
+        id: note.id,
+        title: note.title,
+        description: note.description,
+        isTrash: note.isTrash,
+      };
+      this.listNoteElement.appendChild(this.noteView.constructor.renderNote(noteItem));
+    });
+  };
+
+  /**
+   * @description function render a note
+   *
+   * @param {Object} note is a data of a note
+   */
+  renderNote = (note) => {
+    const noteItem = {
+      id: note.id,
+      title: note.title,
+      description: note.description,
+      isTrash: note.isTrash,
+    };
+    this.listNoteElement.appendChild(this.noteView.constructor.renderNote(noteItem));
+  };
+
+  /**
+   * @description function show input form
+   */
+  showInputForm = () => {
+    this.inputAddElement.addEventListener('focus', () => {
+      this.formUtilitiesElement.style.display = 'block';
+      this.formTitleElement.style.display = 'block';
+    });
+  };
+
+  /**
+   * @description function add a new note and turn off input form
+   *
+   * @param {function} handle is a function transmission in
+   * two String values is title and description of input form
+   */
+  bindAddNewNote = (handle) => {
+    this.closeButtonElement.addEventListener('click', () => {
+      this.titleValue = selectDOMClass('.note-title').value;
+      this.descriptionValue = selectDOMClass('.note-description').value;
+
+      if ((this.titleValue === '') && (this.descriptionValue === '')) {
+        this.formUtilitiesElement.style.display = 'none';
```

Comment:
add a class `.hide` for hidden and reuse then instead of use style attribute.  Then here will become add/remove class